docs/admin/kubeadm.md

```diff
@@ -154,6 +154,11 @@ names `<service_name>.<namespace>.svc.cluster.local`. You can use the
 update the `/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` file
 accordingly else DNS will not function correctly.
 
+**Note: For the time being, this flag will have no effect. Ideally this should
```

Comment:
Suggest something along these lines:

-----

This flag does have effect currently; it's needed for the kube-dns Deployment manifest and the API Server's serving certificate.
However, it's not enough alone if you change it; you have to modify the arguments to the kubelets in the cluster for it to work fully. Rewriting the kubelet's CLI arguments is out of scope for kubeadm as it should be agnostic to how you run the kubelet; however, making all kubelets in the cluster pick up that piece of information anyway dynamically via the API is in scope and a planned feature for current releases, see: [this issue](https://github.com/kubernetes/kubeadm/issues/28)